src/java/io/bazel/rulesscala/scalac/BUILD

```diff
@@ -37,3 +37,14 @@ filegroup(
     ],
     visibility = ["//visibility:public"],
 )
+
+java_test(
+    name = "compile_options_test",
+    srcs = [
+        "CompileOptionsTest.java",
```

Comment:
Maybe we should drop this test? I feel it’s too white box and the other test you added would give the same confidence with more value

Reply:
done.